scene/gui/tab_container.cpp

```diff
@@ -345,7 +345,7 @@ Vector<Control *> TabContainer::_get_tab_controls() const {
 	Vector<Control *> controls;
 	for (int i = 0; i < get_child_count(); i++) {
 		Control *control = Object::cast_to<Control>(get_child(i));
-		if (!control || control->is_set_as_top_level() || control == tab_bar || control == child_removing) {
+		if (!control || control->is_set_as_top_level() || control == tab_bar || (!children_removing.is_empty() && children_removing.has(control))) {
```

Comment:
```suggestion
		if (!control || control->is_set_as_top_level() || control == tab_bar || children_removing.has(control)) {
```

`has`/`find` already do `if (size() == 0) return false;` as a first check, so extra check should not be necessary.